content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -46,7 +46,20 @@ tmpfs, Secrets use a tmpfs, etc.)
 Some popular filesystems that support idmap mounts in Linux 6.3 are: btrfs,
 ext4, xfs, fat, tmpfs, overlayfs.
 
-In addition, support is needed in the 
+In addition, support is needed in the container runtime and in the OCI runtime
+(the low level runtime used by the container runtime).
+
+The OCI runtime requirements are as follows:
+
+ * [crun][crun]: version 1.9 or greater (1.13+ recommended)
+
+Please note that [runc][runc] is the default OCI runtime in several Kubernetes
+distributions. Version `1.1.z` of runc doesn't support all the features needed
+by the Kubernetes implementation. If there is a newer release of runc than 1.1
+available for use, check its documentation and release notes for compatibility
+(look for idmap mounts support in particular, as that is the missing feature).
```

Comment:
Try this wording:
```suggestion

<!-- ideally, update this if a newer minor release of runc comes out, whether or not it includes the idmap support -->
{{< note >}}
Many OCI runtimes do not include the support needed for using user namespaces in Linux pods.
If you use a managed Kubernetes, or have downloaded it from packages and set it up, it's likely
that nodes in your cluster use a runtime that doesn't include this support. For example, the most
widely used OCI runtime is `runc`, and version `1.1.z` of runc doesn't support all the features
needed by the Kubernetes implementation of user namespaces.
If there is a newer release of runc than 1.1 available for use, check its documentation and release
notes for compatibility (look for idmap mounts support in particular, because that is the missing feature).
{{< /note >}}
```

This aligns better with our style guide, and the comment highlights that it's intentionally not timeless (and that's fine; it's a style **guide** not a style rule set). We don't have to update this note every time that there are new runc releases, but we'd do well to maintain it as a best effort.

Reply:
Applied.